packages/swapper/src/swappers/RelaySwapper/utils/types.ts

```diff
@@ -0,0 +1,90 @@
+import type { Execute } from '@reservoir0x/relay-sdk'
+import type { ChainId } from '@shapeshiftoss/caip'
+import type { Address } from 'viem'
+
+import type { TradeQuote, TradeRate } from '../../../types'
+
+export interface RelayTradeQuote extends TradeQuote {
+  selectedRelayRoute?: Execute
+}
+export interface RelayTradeRate extends TradeRate {
+  selectedRelayRoute?: Execute
+}
+
+export type RelayTransactionMetadata = {
+  to: Address | undefined
+  value: string | undefined
+  data: string | undefined
+  gas: string | undefined
+  maxFeePerGas: string | undefined
+  maxPriorityFeePerGas: string | undefined
+}
+
+export type RelayStatus = {
+  status: 'success' | 'failed' | 'pending' | 'refund' | 'delayed' | 'waiting'
+  inTxHashes: string[]
+  txHashes: string[]
+  time: number
+  originChainId: number
+  destinationChainId: number
+}
+
+export type AppFee = {
+  recipient: string
+  fee: string
+}
+
+export type Transaction = {
+  to: string
+  value: string
+  data: string
+}
+
+export type QuoteParams = {
+  user: string
+  originChainId: number
+  destinationChainId: number
+  originCurrency: string
+  destinationCurrency: string
+  tradeType: 'EXACT_INPUT' | 'EXACT_OUTPUT' | 'EXPECTED_OUTPUT'
+  recipient?: string
+  amount?: string
+  txs?: Transaction[]
+  referrer?: string
+  refundTo?: string
+  refundOnOrigin?: boolean
+  useReceiver?: boolean
+  useExternalLiquidity?: boolean
+  usePermit?: boolean
+  useDepositAddress?: boolean
+  slippageTolerance?: string
+  appFees?: AppFee[]
+  gasLimitForDepositSpecifiedTxs?: number
+  userOperationGasOverhead?: number
+  forceSolverExecution?: boolean
+}
+
+export type RelayToken = {
+  chainId: ChainId
```

Comment:
blocking: 

- this is not a CAIP-10 but a Relay internal ChainId
<img width="309" alt="image" src="https://github.com/user-attachments/assets/92eae59c-b7e6-4c23-bdc1-7d4c22369222" />
- This type already exists in the SDK, can we consume it?
https://github.com/reservoirprotocol/relay-kit/blob/4196ea7da82148c27ec1c0f23054aabfe5baa558/packages/sdk/src/types/api.ts#L653-L670

Reply:
I already tried to consume this type, I couldn't achieve it 
![image](https://github.com/user-attachments/assets/09fa919c-6734-4d2b-8cf5-035f3a854e0f)
![image](https://github.com/user-attachments/assets/89d33d87-1a31-4630-972a-731e0f192678)

I took too much time on this and decided to use our own type instead

If they had an exposed type for this, it would have been easier but they don't